python/batch-predictor/docker/process_conda_env.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+CONDA_ENV_PATH="$1"
+
+echo "Processing conda environment file: ${CONDA_ENV_PATH}"
+echo "Current conda environment file content:"
+cat "${CONDA_ENV_PATH}"
+
+# Remove `merlin-sdk` from conda's pip dependencies
+yq --inplace 'del(.dependencies[].pip[] | select(. == "*merlin-sdk*"))' "${CONDA_ENV_PATH}"
+
+# Add `merlin-batch-predictor` to conda's pip dependencies, if not exist
+yq --inplace 'with(.dependencies[].pip; select(all_c(. != "*merlin-batch-predictor*")) | . += ["merlin-batch-predictor==0.40.2.dev29"] )' "${CONDA_ENV_PATH}"
```

Comment:
I guess we'll replace `0.40.2.dev29` with a more 'permanent' (not dev) tag once we're ready to release the changes in this PR? 😮

Reply:
Yes, correct 👍🏻  Will replace it with 0.41.0 once we're ready.